mongo/metadata.csv

```diff
@@ -198,3 +198,21 @@ mongodb.collection.storageSize,gauge,,byte,,Total storage space allocated to thi
 mongodb.collection.nindexes,gauge,,index,,Total number of indices on the collection.,0,mongodb,number of indices on collection
 mongodb.collection.indexSizes,gauge,,byte,,Size of index in bytes.,0,mongodb,index size
 mongodb.collection.indexes.accesses.ops,gauge,,access,,Number of time the index was used.,0,mongodb,index usage
+mongodb.usage.commands.count,rate,,command,,,0,mongodb,
```

Comment:
Can we add even a brief description for these metrics? They go here https://docs.datadoghq.com/integrations/mongo/#data-collected and an empty desc looks bad.